boards/arm/stm32/nucleo-l152re/src/stm32_ili93418b.c

```diff
@@ -0,0 +1,720 @@
+/****************************************************************************
+ * boards/arm/stm32/nucleo-l152re/src/stm32_ili93418b.c
+ *
+ * Driver for the ILI9341 Single Chip LCD driver connected
+ * via 8080-I 8-bit Series Parallel Interface
+ *
+ *   Author: Chunyang Liu <yjdwbj@gmail.com>
+ *
+ * References: ILI9341_DS_V1.10.pdf (Rev: 1.10), "a-Si TFT LCD Single Chip
+ *             Driver 240RGBx320 Resolution and 262K color", ILI TECHNOLOGY
+ *             CORP., http://www.ilitek.com.
+ *             ILI TECHNOLOGY CORP., http://www.ilitek.com.
```

Comment:
it's better to move the reference section out of copyright header, so the tool can identify the copyright more easier.